rust/hyperlane-base/src/db/rocks/hyperlane_db.rs

```diff
@@ -164,10 +140,10 @@ impl HyperlaneRocksDB {
             return Ok(false);
         }
         // Set the gas payment as processed
-        self.store_gas_payment_meta_processed(&payment_meta)?;
+        self.store_processed_by_gas_payment_meta(&payment_meta, &true)?;
```

Comment:
Drive by request: fix `store_processed_by_gas_payment_meta` to take `bool` instead of `&bool`

Reply:
I'm not sure there's an easy way to do this?